docs/readthedocs/source/doc/Nano/QuickStart/tensorflow_quantization_quickstart.md

```diff
@@ -76,9 +76,14 @@ model.fit(train_ds, epochs=1)
 ```python
 from tensorflow.keras.metrics import CategoricalAccuracy
 q_model = InferenceOptimizer.quantize(model,
-                                      calib_dataset=dataset,
+                                      x=dataset,
+                                      precision='bf16',
+                                      accelerator='openvino',
                                       metric=CategoricalAccuracy(),
-                                      tuning_strategy='basic'
+                                      accuracy_criterion={'relative': 0.01, 'higher_is_better': True},
+                                      tuning_strategy='basic',
+                                      timeout=0,
+                                      max_trials=10,
```

Comment:
to be honest, what should we do to https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/index.html? @rnwang04 @MeouSker77 . We will meet more and more not up-to-date information for these pages and we seems to suggest our users to only read key features + how to guide.

Reply:
Got your points~